esx_service/cli/vmdkops_admin.py

```diff
@@ -857,8 +859,8 @@ def tenant_rm(args):
     # If args "remove_volumes" is not specified in CLI
     # args.remove_volumes will be None
     if args.remove_volumes:
-        if args.remove_volumes == 'True':
-            remove_volumes = True
+        print "All Volumes will be removed"
```

Comment:
it's OK. I will change others in one pass for P3 move (using 2x3 tool). I just want to avoid new ones popping up